### PR TITLE
create advisory board pages on csedweek and hoc.com

### PR DESCRIPTION
**Pull request by tanyaparker**
csedweek.org/about/advisory-board
image

hourofcode.com/advisory-board
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
@breville unfortunately had to copy code.org views since hoc.com doesn't inherit. Basically like how we copied the partners page.

File: pegasus/sites.v3/hourofcode.com/views/about_headshots.haml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#headshots{:style=>'width: 900px; padding: 0; margin: 0;'}
+  - people.each_slice(columns) do |group|
+    .headshot_row{:style=>'width: 900px; padding: 0; margin: 0;'}
+      - group.each do |person|
+        - column_width = columns==3 ? '300' : '160'
+        .headshot_column{:style=>'width: ' + column_width + 'px; padding: 15px; margin: 0; float: left;'}
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/views/about_headshots.haml:6]: Obvious improvements could be made here, like extracting CSS, but understood if you just want to make this a duplicate.

**Comment by breville** [comment]
Some people seem to have two photos in different directories.. deliberate?

**Comment by tanyaparker** [comment]
The images are copied too in code.org/public/images/avatars and hourofcode.com/public/images. We made a decision before to copy. The problem would be if we have them all pull from code.org, and someone deletes them from code.org and doesn't realize that hoc.com also uses it.

**Comment by breville** [comment]
lgtm

**Pull request merged**